Testing/Data/Input/ThresholdSegmentationLevelSetVentricle.png.sha512

```diff
@@ -0,0 +1 @@
+4a75c62c8b9778683f39eafcd31fc4274fcb7c9096d4cbbbb1f798da5a37becc5657a24d24092f887c1919dfade1aa9964399cf982ace4a3b79865c501b8d746
```

Comment:
If you open this file in a text editor, you should be able to hit enter and the end of the line add a new line there.